pool/app/message_template/message_utils.ml

```diff
@@ -155,17 +155,20 @@ let message_langauge sys (contact : Contact.t) =
   |> CCResult.pure
 ;;
 
-let search_by_language templates language =
-  let open CCResult in
+let search_by_language templates lang =
   let open Pool_common in
-  CCList.find_opt (fun t -> t |> fst |> Language.equal language) templates
+  CCList.find_opt
+    (fun { language; _ } -> Language.equal language lang)
+    templates
+  |> (function
+       | None -> templates |> CCList.head_opt
+       | Some template -> CCOption.pure template)
```

Comment:
Please use either `Some` or `CCOption.return` to use consistent API (not the Applicative one)